### PR TITLE
Add ssl option on login

### PR DESCRIPTION
**Pull request by aliicewong**
- Create checkbox for SSL in the UI 
- Add functionality to add ?ssl=true if the checkout is checked when the user selects "login"
- Not pretty but functional :)

### CODE REVIEW

File: app/components/Login.tsx [hunk 1]
```diff
@@ -68,17 +72,19 @@ const RequiredWarning = styled.span`
 `
 
 const Login = () => {
-  const [URI, setURI] = useState('');
+  const [ URI, setURI ] = useState('');
```
Comment (saylestyler) [inline app/components/Login.tsx]: we should pick one or the other spacing styles `[ s, s1 ]` | `[s, s1]`

File: app/components/Login.tsx [hunk 2]
```diff
@@ -68,17 +72,19 @@ const RequiredWarning = styled.span`
...
   const [redirectToHome, setRedirectToHome] = useState(false);
 
   const sendLoginURI = (): void => {
-    // #TODO: toggleSSL = add ?ssl=true
-    // let useSSL: boolean = false;
-    // const toggleSSL = (): boolean => useSSL = true;
-    const client = new Client(URI + '?ssl=true')
+
+    let updatedURI = URI;
+    if ( isSSL ) updatedURI += '?ssl=true';
```
Comment (saylestyler) [inline app/components/Login.tsx]: ditto spacing here

File: app/components/Tables.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ const TableRow = styled.li`
   border: ${ ({affected}) => affected ? '3px solid pink' : '3px solid transparent'};
   border-top: 1px solid lightgrey;
   transition: 0.3s;
-
```
Comment (saylestyler) [inline app/components/Tables.tsx]: extra newline

File: app/components/Login.tsx (hunk 1 above)
Comment (aliicewong) [reply app/components/Login.tsx]: I like the ones that have the space on the inside. What do you guys think? @AHyman18 @katamhsw @saylestyler

Comment (saylestyler) [reply app/components/Login.tsx]: i'd say none, not bc i think it looks better but bc it's what react uses in [their docs](https://reactjs.org/docs/hooks-intro.html)

Comment (aliicewong) [reply app/components/Login.tsx]: that works for me too, ill update 👍

File: app/components/Tables.tsx (hunk 3 above)
Comment (aliicewong) [reply app/components/Tables.tsx]: updated

File: app/components/Login.tsx (hunk 2 above)
Comment (aliicewong) [reply app/components/Login.tsx]: updated

**Pull request merged**